### PR TITLE
fix: updating a non-existing property will add a new property to the …

### PR DESCRIPTION
**Pull request by xzdry**

### CODE REVIEW

File: packages/editor/__tests__/model/componentModel.spec.ts [hunk 1]
```diff
@@ -33,12 +33,17 @@ describe('update component property', () => {
   const origin = appModel.toSchema();
   const text1 = appModel.getComponentById('text1' as any);
   text1!.updateComponentProperty('value', { raw: 'hello', format: 'md' });
+  text1!.updateComponentProperty('newProperties', "a property that didn't exist brefore");
```
Comment (tanbowensg) [inline packages/editor/__tests__/model/componentModel.spec.ts:36]: brefore => before

File: packages/editor/__tests__/model/componentModel.spec.ts [hunk 2]
```diff
@@ -33,12 +33,17 @@ describe('update component property', () => {
...
   const newSchema = appModel.toSchema();
 
   it('update component properties', () => {
     expect(newSchema[5].properties.value).toEqual({ raw: 'hello', format: 'md' });
   });
 
+  it("update a new commponent that didn't exist",()=>{
```
Comment (tanbowensg) [inline packages/editor/__tests__/model/componentModel.spec.ts:43]: commponent => property

**Review by tanbowensg — COMMENTED** [review]
some typo

**Comment by xzdry** [comment]
> some typo

already corrected

**Review by tanbowensg — APPROVED** [review]

**Pull request merged**